### ISSUE TITLE
Maven complains on guava version in 0.8.2

### ISSUE THREAD

**Issue by ssardina** [sarl/sarl#867]
**Describe the bug**

When I migrated 0.7.2 to 0.8.2 I get this in CLI compile with maven:

```
[ERROR] Unexpected version for the artifact com.google.guava:guava.
Minimal expected version: 20.0
Current version: 19.0
You must select the appropriate version in the dependency hierarchy, by updating the project configuration with:
<dependencyManagement>
	<dependencies>
		<dependency>
			<groupId>com.google.guava</groupId>
			<artifactId>guava</artifactId>
			<version>20.0</version>
		</dependency>
	</dependencies>
</dependencyManagement>

```

If I add that to the `pom.xml` it works, but I guess this should not be necessary

Code compiles with no error under Eclipse though...

**Expected behavior**

No error, like it was in 0.7.2

**System configuration:**
-- SARL version:	
-- SARL compiler:	
-  - [] Eclipse compiler without Maven	
-  - [ ] Eclipse compiler with Maven	
-  - [X] Maven compiler on the command line	
-  - [ ] sarlc compiler	
-- Java JDK version (with the manufacturer name):	
-- Operating System:	
-  - [x] Linux 64bits	
-  - [ ] Linux 32bits	
-  - [ ] Windows 64bits	
-  - [ ] Windows 32bits	
-  - [ ] MacOS 64bits

**Comment by gallandarakhneorg** [comment]
I cannot reproduce the problem with a base Maven project.
The problem may be cause by a higher priority dependency that depends on Guava 19.0.
Could you provide to me a screenshot your the "Maven Dependency" tab within the pom editor of Eclipse. Search for "guava" in this tab.
An example of screenshot that I have obtained is attached:
![capture du 2018-10-03 06-39-44](https://user-images.githubusercontent.com/4200154/46390324-93847580-c6d7-11e8-8ef7-253145bc3d6d.png)

**Comment by gallandarakhneorg** [comment]
I found the source of the problem: the invalid version of Guava comes from the Janus Kernel itself, not from the SARL libraries. I will fix it for version `0.8.3`

**Comment by ssardina** [comment]
Great, thanks!
